### PR TITLE
Improve the WebApp manifest generated by Webpack

### PR DESCRIPTION
**Pull request by TomTirapani**
Deliver a better user experience when using the 'Add to Homescreen' shortcut on mobile devices.

### CODE REVIEW

**Comment by TomTirapani** [comment]
Further info:

https://developer.mozilla.org/en-US/docs/Web/Manifest
https://github.com/itgalaxy/favicons#usage

File: configureWebpack.js [hunk 1]
```diff
@@ -584,6 +584,10 @@ async function configureWebpack(env) {
                 inject: true,
                 mode: 'webapp',
                 favicons: {
+                    appName: appName,
+                    theme_color: '#000',
+                    appleStatusBarStyle: 'black',
+                    start_url: '/mobile',
```
Comment (TomTirapani) [inline configureWebpack.js:590]: Do we want to assume that most apps will use this url for their mobile app? The "Add to Homescreen" option sets the url to domain + start_url, and does not allow the user to change it. Without it, we're relying on nginx or something doing the redirect.

Comment (amcclain) [reply configureWebpack.js:590]: Good question - I think we might as well go ahead and make it configurable as a top-level arg to this method, with a default (that we expect to be used for almost all apps) of `mobile`.  

Could call the arg `defaultMobilePath`.  Open to other names - we have `devServerOpenPage` with that term `page` - but that's related to the specific webpack config and seems a bit weird to use here.

Probably less surprising to have the code inside this method prepend the leading `/` so the dev doesn't need to.

**Comment by amcclain** [comment]
Looking at [other options in the docs here](https://github.com/itgalaxy/favicons#usage) - I've been meaning to look into the chrome warning re. "Manifest: property 'description' ignored, type string expected."

Can we pls. also try setting `appDescription` - could simply repeat app name - to see if that squelches warning.  Not convinced we need to make this a configurable thing as well, but would be nice to skip the console warn on every page load if that does it

**Comment by amcclain** [comment]
Also wonder if there is any advantage to setting `appVersion` - we know that, so why not

**Comment by TomTirapani** [comment]
Just made those changes - will deploy to confirm squelching the console warning.

Can confirm so far that the shortcut name and status bar colors have been improved. Before:
Before
After:
After

**Comment by TomTirapani** [comment]
Can confirm that the console warning no longer shows after providing `appDescription`

**Pull request merged**